lib/data/network/dio_cache_option.dart

```diff
@@ -24,7 +25,7 @@ class DioCacheOption {
     if (kIsWeb) return null;
     final appDir = await getApplicationDocumentsDirectory();
     Logs().d('DioCacheOption::_getAppDirPath() appDirPath ${appDir.path}');
-    return appDir.path;
+    return '${appDir.path}$_dioCacheFolder';
```

Comment:
https://pub.dev/documentation/path_provider/latest/path_provider/getApplicationDocumentsDirectory.html
getApplicationDocumentsDirectory can throw exception , please add try catch for that

Reply:
done